### PR TITLE
Translate pipeling-settings-en doc-3.1

### PR DESCRIPTION
**Pull request by serenashe**

### CODE REVIEW

**Comment by Felixnoo** [comment]
Please add updates once #1509 gets merged, thanks.

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-settings.md [hunk 1]
```diff
@@ -1,199 +1,199 @@
...
 
   {{</ tab >}}
 
   {{< tab "GitLab" >}}
 
-  ![code-source-gitlab](/images/docs/devops-user-guide/using-devops/pipeline-settings/code-source-gitlab.png)
+  ![code-source-gitlab1](/images/docs/zh-cn/devops-user-guide/use-devops/pipeline-settings/code-source-gitlab1.png)
 
-  If you select **GitLab**, you have to specify the GitLab server, owner and repository name. You also need to specify a credential if it is needed for obtaining repository codes. Click the **√** icon after you finish all operations.
+  如果选择 **GitLab**，则必须指定 GitLab 服务、项目所属组和仓库名称。如果获取仓库代码需要凭证，则需要指定一个凭证。完成所有操作后，记得点击 **√** 图标。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-settings.md:46]: 服务 ->  Server
项目所属组  ->  项目 Owner

**Review by Felixnoo — CHANGES_REQUESTED** [review]

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-settings.md (hunk 1 above)
Comment (serenashe) [reply content/zh/docs/devops-user-guide/how-to-use/pipeline-settings.md:46]: These two terms were translated according to the Chinese UI, please take a look at the screenshot I put in this file, thanks!

Comment (Felixnoo) [reply content/zh/docs/devops-user-guide/how-to-use/pipeline-settings.md:46]: Got it. Ignore my comment.

**Pull request merged**